pkgs/by-name/xd/xdg-desktop-portal-luminous/package.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  meson,
+  ninja,
+  libclang,
+  clang,
+  xdg-desktop-portal,
+  slurp,
+  cairo,
+  pango,
+  libxkbcommon,
+  glib,
+  pipewire,
+}:
+
+rustPlatform.buildRustPackage rec {
```

Not strictly necessary, but could use `finalAttrs` pattern instead for easier overriding

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```